src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -2107,6 +2107,20 @@ For more information, see https://github.com/ow2-proactive/scheduling/blob/maste
 === Catalog
 include::./Catalog.adoc[]
 
+[[_backup]]
+== Configure automated backup
+
```

Comment:
you need to introduce the automated backup mechanism's scope:
We provide an internal backup mechanism for the database, in case HSQLDB is used, and for data files (workflows, logs).
In case another DB is used, it is possible to combine a vendor specific backup mechanism with the internal mechanism for data files.

Reply:
ok